projects/sbb-esta/angular-business/status/src/status/status-type.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum StatusType {
```

Comment:
I don't think we have something like this for other modules. Is there a specific reason why you added this?